onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListFragment.java

```diff
@@ -1638,6 +1639,23 @@ public boolean equals(Object obj) {
         }
     }
 
+    private void reportAnalytics(ObaSituation situation) {
+        ObaSituation.AllAffects[] allAffects = situation.getAllAffects();
+        List<String> agencyIds = new ArrayList<>();
+        for (int i=0; i< allAffects.length; i++) {
+            ObaSituation.AllAffects affect = allAffects[i];
```

Comment:
I believe you can get rid of the declaration of `ObaSituation.AllAffects affect` here, and just reference `allAffects[i]` directly in the following lines of code?

So it would change to:

~~~
if (!agencyIds.contains(allAffects[i].getAgencyId())) {
    agencyIds.add(allAffects[i].getAgencyId());
}
~~~

Also, minor formatting issue - in the IF block, there should be a space between `) {` - I fixed that in the above example.